aiidalab/git_util.py

```diff
@@ -21,14 +35,14 @@ class BranchTrackingStatus(Enum):
     DIVERGED = 2
 
 
-class GitManagedAppRepo(Repo):
+class GitManagedAppRepo(Repo):  # type: ignore
     """Utility class to simplify management of git-based apps."""
 
-    def list_branches(self):
+    def list_branches(self):  # type: ignore
```

Comment:
```suggestion
    def list_branches(self) -> Any: 
```
Not sure this is good, we know it has a return value, so maybe just return Any?